### ISSUE TITLE
Support Android File Chooser (to be able to add rclone remote) 

### ISSUE THREAD

**Issue by anaxonda** [FossifyOrg/Gallery#201]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

The rclone management app [roundsync](https://github.com/newhinton/Round-Sync) supports exposing rclone remotes to other apps, as a local provider. 
For example, in my file manager [Material Files](https://github.com/zhanghai/MaterialFiles) I can add a Backblaze (or nearly any other cloud storage) hosted folder, and manage files and directories as I would with local files. 

However, if the Gallery app uses its own directory chooser when choosing which folders to show as albums. In that interface, only folders on the SD card are shown, so I cannot choose my rclone directory.

### Why do you want this feature?

This would enable viewing remote, cloud hosted photo albums. I'm looking for a solution to replace google photos, to be able to access my photo library (there are other nice solutions, eg Immich and Photoprism however they require servers.)

### Additional information

I'm not sure if enabling using just the stock "file-picker" would be enough to enable showing these directories, but if so it should be pretty easy to enable.

**Comment by Philippe-Choquette** [comment]
It would also be useful to have this option when you want to move a photo to another folder (Move to > Other folder) in order to save into mounted cloud paths like Google Drive or else.

**Comment by Philippe-Choquette** [comment]
@Aga-C Are there plans to improve the file picker? It's the same file picker that is shared by all your apps, snd it's the only limitation that's a deal-breaker for me.

**Comment by naveensingh** [comment]
Yeah, we _might_ drop the custom file picker in favor of the Android file picker.

**Comment by Overexcited** [comment]
Personally, I like the aesthetics and efficient flow of the built-in Fossify file picker, but I agree it would also be good to have access to the native Android file picker too, such that users can use special locations such as cloud storages added as a document provider, etc.

Therefore I suggest the best middle road would be to leave the current built-in file picker intact, and simply have the "Other" button open the stock Android file picker. Best of both worlds.

**Comment by naveensingh** [comment]
I like it too, but it's best to remove it. There's a related issue affecting other Fossify apps: https://github.com/FossifyOrg/General-Discussion/issues/698

**Comment by aroguecoder** [comment]
watching this issue, don't mind me
